streaming/BUILD.bazel

```diff
@@ -143,6 +152,60 @@ cc_library(
     }),
 )
 
+cc_library(
+    name = "streaming_channel",
+    srcs = glob(["src/channel/*.cc"]),
+    hdrs = glob(["src/channel/*.h"]),
+    includes = ["src/channel/"],
+    deps = [
+        ":streaming_util",
+        ":streaming_queue",
+        ":streaming_message",
+        ":streaming_ring_buffer",
+        ":streaming_common",
+    ],
+    copts = COPTS,
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "streaming_reliability",
+    srcs = glob([ "src/reliability/*.cc" ]),
+    hdrs = glob([ "src/reliability/*.h"]),
+    includes = ["src/", "src/reliability/"],
+    deps = [
+        ":streaming_util",
+        ":streaming_channel",
+        ":streaming_message",
+    ],
+    copts = COPTS,
+    visibility = ["//visibility:public"],
+)
+
+cc_library(
+    name = "streaming_ring_buffer",
+    srcs = glob([ "src/ring_buffer/*.cc" ]),
+    hdrs = glob([ "src/ring_buffer/*.h"]),
+    includes = ["src/", "src/ring_buffer/"],
```

Comment:
Include like `src/ring_buffer/` should be avoid, and all include code in `.cc/.h` should has a more qualified path, task ray as an example: `#include ray/core_worker/core_worker.h`

Reply:
done.